promise-types/json/json_promise.py

```diff
@@ -0,0 +1,75 @@
+from cfengine import PromiseModule, ValidationError, Result
+from typing import Any
+import json
+import os
```

Comment:
Very small nitpick, but it's relatively common to order / group your imports, so standard library is first, etc.

```suggestion
import os
import json
from typing import Any

from cfengine import PromiseModule, ValidationError, Result
```